examples/.env.test

```diff
@@ -1,2 +1,2 @@
-AUTO_DRIVE_API_KEY=
+AUTO_DRIVE_API_KEY=97cc977a535f4d89a3369ad5404f8ba0
```

Comment:
This seems like a leak key, if it's a valid key, we should revoke it

Reply:
Thanks for the catch, was a local API_KEY so no sec problem.